profiles-sdk-examples/docs/config.md

```diff
@@ -0,0 +1,118 @@
+# Configuration Options
+
+Cortex configuration properties can be specified in the [Spark Configurations](https://spark.apache.org/docs/latest/index.html).
+
+Configuration options are specific to your use case and may depend on:
+- How/where you intend to run your application?
+- What resources are available in the target environment?
+- What your use case is?
+
+<-- TODO: General Spark configuration guidance (Spark UI, Debugging, metrics, fs performance)
```

Comment:
did you want this commented out?  you need a exclamation point after the lt sign